cardano-cli/test/cardano-cli-golden/Test/Golden/Governance/DRep.hs

```diff
@@ -8,13 +8,10 @@ module Test.Golden.Governance.DRep where
 #endif
 
 import           Control.Monad
-
-#ifdef UNIX
```

Comment:
Is this change intended? If yes can you explain?

Reply:
That's most likely stylish-haskell. It's support for CPP is buggy.